src/python/sdk/examples/trade.py

```diff
@@ -0,0 +1,533 @@
+    )
+    place_limit_orders_at_market(
+        viewer, account_A, market_id, 200, buy_ticks_per_lot, sell_ticks_per_lot
+    )
+    place_limit_orders_at_market(
+        viewer, account_A, market_id, 300, buy_ticks_per_lot, sell_ticks_per_lot
+    )
+    place_limit_orders_at_market(
+        viewer, account_A, market_id, 400, buy_ticks_per_lot, sell_ticks_per_lot
+    )
+    place_limit_orders_at_market(
+        viewer, account_A, market_id, 500, buy_ticks_per_lot, sell_ticks_per_lot
+    )
+    print("Account A has created multiple competitive limit orders!")
+
+    report_best_price_levels(viewer, market_id)
+
+    if start_ask_price == None:
+        exit()
+
+    input("\nPress enter to place spread-crossing limit orders with Account B.")
+    fills = get_fill_events(viewer, account_B.account_address, market_id, 0)
+    filled_size_pre = len(fills)
+    volume = 100 + 200 + 300 + 400 + 500
+    place_limit_order(Side.ASK, account_B, market_id, volume, buy_ticks_per_lot)
+    fills = get_fill_events(viewer, account_B.account_address, market_id, 0)
+    filled_size_ask = len(fills)
+    print(
+        f"  * There were {filled_size_ask-filled_size_pre} BID orders filled by the ASK order placement."
+    )
+    place_limit_order(
+        Side.BID, account_B, market_id, volume * 2, start_ask_price
+    )
+    fills = get_fill_events(viewer, account_B.account_address, market_id, 0)
+    filled_size_bid = len(fills)
+    print(
+        f"  * There were {filled_size_bid-filled_size_ask} ASK orders filled by the BID order placement."
+    )
+    print("Account B has finished placing 2 cross-spread limit orders.")
+    mkt_account = get_market_account(
+        viewer, account_B.account_address, market_id, 0
+    )
+    mkt_account_bid_count = len(mkt_account["bids"])
+    mkt_account_ask_count = len(mkt_account["asks"])
+    print(
+        f"  * {mkt_account_bid_count} BID and {mkt_account_ask_count} ASK orders remain open on the account."
+    )
+    report_best_price_levels(viewer, market_id)
+
+
+def place_market_order(
+    direction: Side,
+    account: Account,
+    market_id: int,
+    size_lots_of_base: int,
+):
+    calldata = place_market_order_user_entry(
+        ECONIA_ADDR,
+        TypeTag(StructTag.from_str(COIN_TYPE_ETH)),
+        TypeTag(StructTag.from_str(COIN_TYPE_USDC)),
+        market_id,
+        AccountAddress.from_hex("0x0"),
+        direction,
+        size_lots_of_base,
+        SelfMatchBehavior.CancelMaker,
+    )
+    EconiaClient(NODE_URL, ECONIA_ADDR, account).submit_tx_wait(calldata)
+
+
+def place_limit_order(
+    direction: Side,
+    account: Account,
+    market_id: int,
+    size_lots_of_base: int,
+    price_ticks_per_lot: int,
+):
+    calldata = place_limit_order_user_entry(
+        ECONIA_ADDR,
+        TypeTag(StructTag.from_str(COIN_TYPE_ETH)),
+        TypeTag(StructTag.from_str(COIN_TYPE_USDC)),
+        market_id,
+        AccountAddress.from_hex("0x0"),
```

Comment:
Can you change the integrator address here and for the market account to be the Econia address?

I'd like for the tutorial follower to understand that under the hood Econia is going to collect taker fees and that will make it explicit

Reply:
will do